Dockerfile

```diff
@@ -1,24 +1,22 @@
-FROM golang:1.17
+FROM golang:1.19
 
 ENV CGO_ENABLED=0
 
-# install tooling
-RUN go get -v \
-  github.com/kisielk/errcheck \
-  golang.org/x/lint/golint
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  ca-certificates \
+  git \
+  unzip
+
+RUN update-ca-certificates
```

Comment:
nit: if we're using recent Golang base images, it should already contain sufficiently recent CA certs

Reply:
Turns out we don't need any of those dependencies.